### PR TITLE
[RFR] Fix model tests so that they ... work

### PR DESCRIPTION
**Pull request by cwisecarver**
There are some changes that will be in a separate PR to OSF @sloria but this should fix the osf_models side of the tests.

### CODE REVIEW

**Comment by sloria** [comment]
Cool. Please send the OSF updates to COS/feature/django-osf

File: osf_models/models/sanctions.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class Sanction(ObjectIDMixin, BaseModel):
     mode = UNANIMOUS
 
     # Sanction subclasses must have an initiated_by field
-    # initiated_by = fields.ForeignField('user', backref='initiated')
+    initiated_by = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True)
```
Comment (sloria) [inline osf_models/models/sanctions.py]: I don't think this should actually be here; it's commented out in v1. IIRC, `EmbargoTerminationApproval`s don't have an initiated_by field.

Comment (cwisecarver) [inline osf_models/models/sanctions.py]: Hmm. I was trying to fix this: https://github.com/CenterForOpenScience/osf-models/blob/master/osf_models/models/registrations.py#L263

**Comment by cwisecarver** [comment]
Partner PR on OSF: https://github.com/CenterForOpenScience/osf.io/pull/6370

File: osf_models/models/sanctions.py (hunk 1 above)
Comment (sloria) [inline osf_models/models/sanctions.py]: Fair enough. Indeed, Django is more strict than ODM about passing extra arguments to a model constructor.

I'm not completely sure whether or not we intentionally left the field out, but FWIW, there are no EmbargoTerminationApprovals that have `initiated_by` on prod

```python
In [2]: models.EmbargoTerminationApproval.find(Q('initiated_by', 'ne', None)).count()
Out[2]: 0
```

Comment (sloria) [inline osf_models/models/sanctions.py]: Maybe we just keep like it is in v1 (commented out), then remove the invalid argument you linked to above.

Comment (cwisecarver) [inline osf_models/models/sanctions.py]: Sounds good to me.

Comment (cwisecarver) [inline osf_models/models/sanctions.py]: Thanks. I was about to fix it until I pulled.

**Comment by sloria** [comment]
Could you take out the print statements here in this PR? https://github.com/CenterForOpenScience/osf-models/blob/0a071a6c6abc8c979a00bb586ca713a36110702e/osf_models/models/base.py#L477

File: osf_models/models/sanctions.py (hunk 1 above)
Comment (sloria) [inline osf_models/models/sanctions.py]: Sounds good. Once this change is reverted, this is good to merge.

**Pull request merged**